### PR TITLE
Ubuntu/impish: Add Depends: python3-debconf

### PR DESCRIPTION
**Pull request by aciba90**
Depends: python3-debconf added in https://github.com/canonical/cloud-init/pull/1506 and https://github.com/canonical/cloud-init/pull/1472

SC-1102

(Do not squash)

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+cloud-init (22.2-0ubuntu1~21.10.1) UNRELEASED; urgency=medium
```
Comment (blackboxsw) [inline debian/changelog:1]:
```suggestion
cloud-init (22.2-0ubuntu1~21.10.2) UNRELEASED; urgency=medium
```

File: debian/control [hunk 2]
```diff
@@ -38,6 +38,7 @@ Depends: cloud-guest-utils | cloud-utils,
          netplan.io,
          procps,
          python3,
+         python3-debconf,
```
Comment (blackboxsw) [inline debian/control:41]: We also need this in Build-Depends too. See https://github.com/canonical/cloud-init/pull/1511 for context.

**Review by blackboxsw — CHANGES_REQUESTED** [review]
Thanks @aciba90  for the pkging branches. Same comments as other PRs.

**Comment by aciba90** [comment]
Thanks for the review @blackboxsw. I have applied the requested changes. Ready to be reviewed.

**Pull request merged**